packages/type-extrinsics/src/fromMetadata/index.ts

```diff
@@ -21,12 +21,12 @@ export default function fromMetadata (metadata: Metadata): ModulesWithMethods {
   return metadata.asV4.modules
     .filter((modul) => modul.calls.isSome)
```

Comment:
Unfortunately delayed nitpick: We have a quite german name for `module` here . Don't know if it's worth another PR, but I'd rename `modul` to the english `module`

Reply:
module is a reserved name in js, I renamed to modul here (and in multiple other places) on purpose

Reply:
Ah, ok, makes sense. it looked liked an oversight to me.

Reply:
Couple of these where we deviate from Rust -

- `arguments` -> `args`
- `call` -> `method`
- `module` -> `modul` (only for loops on decorating, it not exposed anywhere by this name)

Reply:
Come to think about it: `type` is a reserved TS word. Though it seems to work okay `const { type } = a;` everywhere, so we don't need to change it.

Reply:
I would hate `something.typ` as a getter :)

Reply:
I think rust uses `.ty`, but not better

Reply:
Rust also has `typ` instead of `type`? There really were some germans involved in developing that language 😄 

These two make sense
```
arguments -> args
call -> method
```